http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/HttpRoutesSpec.scala

```diff
@@ -227,23 +227,13 @@ class HttpRoutesSpec
     }
   }
 
+  // Tests only parameter parsing, not service logic.
```

Comment:
I don't see the utility of a method which purports to "end to end test" an endpoint but uses entirely mocked dependencies returning dummy data; these effects are better tested in the smaller components or from outside as a real integration test